packages/plugins/typescript/operations/src/ts-selection-set-processor.ts

```diff
@@ -23,6 +24,10 @@ export class TypeScriptSelectionSetProcessor extends BaseSelectionSetProcessor<S
         useTypesPrefix: true,
       });
 
+    if (unsetTypes) {
+      return [`MakeEmpty<${parentName}, ${fields.map(field => `'${field.fieldName}'`).join(' | ')}>`];
```

Comment:
I wonder, whether we want to use the helper type here (`MakeEmpty`), or instead just hard-code the fields in an inline type. In the past we used type mappers a lot and we decided against keeping the generated output as raw and explicit as possible, even if it would mean more characters.

Reply:
I added it for the `preResolveTypes: false` case because I figured that it should use mappers in this case. For `preResolveTypes: true` it's inlined.

Reply:
Great! We want to remove `preResolveTypes: false` at some point